doc/UsersGuide/source/CodeOverview.rst

```diff
@@ -1,14 +1,48 @@
 .. _CodeOverview:
 
 *************************
-Code Overview
+Technical Overview
 *************************
 
+.. _SupportedPlatforms:
+
+=====================================================
+Supported Systems for Running the UFS Weather Model
+=====================================================
+
+Four levels of support have been defined for :term:`UFS` applications, and the UFS Weather Model (:term:`WM`) operates under this paradigm: 
+
+* **Level 1** *(Preconfigured)*: Prerequisite software libraries are pre-built and available in a central location; code builds; full testing of model.
+* **Level 2** *(Configurable)*: Prerequisite libraries are not available in a centralized location but are expected to install successfully; code builds; full testing of model.
+* **Level 3** *(Limited-test platforms)*: Libraries and code build on these systems, but there is limited testing of the model.
+* **Level 4** *(Build-only platforms)*: Libraries and code build, but running the model is not tested.
+
+Level 1 Systems
+==================
+Preconfigured (Level 1) systems for the UFS WM already have the required external libraries available in a central location via :term:`HPC-Stack` or :term:`spack-stack`. The WM is expected to build and run out-of-the-box on these systems, and users can download the WM code without first installing prerequisite software. In general, users must have access to these Level 1 systems in order to use them.
+
+Currently, Level 1 (or Tier-1) platforms for regression testing are: 
+
+   * WCOSS2 (Intel)
+   * Cheyenne (Intel/GNU compilers)
+   * Gaea (Intel)
+   * Hera (Intel/GNU compilers)
+   * Jet (Intel)
+   * Orion (Intel)
+   * AWS Docker container (Intel)
+
+More information is available in the `UFS WM wiki <https://github.com/ufs-community/ufs-weather-model/wiki/Regression-Test-Policy-for-Weather-Model-Platforms-and-Compilers>`__. 
+
+Level 2-4 Systems
```

Comment:
Could you add S4 (Intel) as a supported level 2 platform?

Reply:
I think that is up to @jkbk2004 and the rest of the WM Team. I have no objection, but it's not my call. ;)